packages/Slider/styles.ts

```diff
@@ -0,0 +1,192 @@
+import styled, { css, th } from '@xstyled/styled-components'
+import { Box } from '@welcome-ui/box'
+import { Hint } from '@welcome-ui/hint'
+
+type BorderProps = {
+  borderSelectorColor?: string
+}
+
+export const Slider = styled.input<BorderProps>(
+  ({ borderSelectorColor, disabled }) => css`
```

Comment:
Shouldn't we giving a default value for borderSelectorColor as it's not mandatory?

Reply:
The issue I see here is that if borderSelectorColor is undefined, you will have `border-color: undefined`.
It will cause an error in your console I think. 
If you set borderSelectorColor = transparent by default, it will not override the border-color provided when borderSelectorColor is undefined, and prevent any console error. 
As BorderProps' borderSelectorColor is not required, it's possible to receive undefined as a value I guess.